.kitchen.yml

```diff
@@ -11,6 +11,9 @@ platforms:
   - name: ubuntu-12.04
   - name: debian-6.0.8
   - name: centos-6.4
+    run_list:
+      - recipe[yum-epel::default]
+      - recipe[yum-remi::default]
```

Comment:
I agree. We shouldn't force the user to accept a particular repo or update timing (when apt-get update is run, for example).